templates/chess.html

```diff
@@ -7,88 +7,25 @@
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-        </tr>
-        <tr class="row">
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-        </tr>
-        <tr class="row">
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-        </tr>
-        <tr class="row">
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-        </tr>
-        <tr class="row">
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-        </tr>
-        <tr class="row">
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-        </tr>
-        <tr class="row">
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-            <td class="square"></td>
-        </tr>
+        {% for i in range(0, 8)%}
+            <tr class="row">
+            {% for j in range(0, 8)%}
+               {% if i % 2 == 0 %}
+                  {% if j % 2 == 0 %}
+                    <td class="square even"></td>
+                  {% else %}
+                    <td class="square odd"></td>
+                  {% endif %}
+               {% else %}
+                    {% if j % 2 == 0 %}
+                        <td class="square odd"></td>
+                    {% else %}
+                        <td class="square even"></td>
+                    {% endif %}
+               {% endif %}
```

Comment:
You have only 2 exits to the if statement, but have written it out as 4. You should be able to condense these to be simpler.